src/utils/tokens.ts

```diff
@@ -66,6 +66,28 @@ export const getTokenMap = async (
     }
   }
 
+  const rawSaberTokenList = (
+    await axios.get<any>(
+      "https://registry.saber.so/data/token-list.mainnet.json"
+    )
+  ).data.tokens as TokenInfo[];
```

Comment:
Instead of casting we should type it with axios, think it's `{ tokens: TokenInfo[] }`